src/compiler/types.ts

```diff
@@ -109,6 +109,8 @@ export const enum SyntaxKind {
     BacktickToken,
     /** Only the JSDoc scanner produces HashToken. The normal scanner produces PrivateIdentifier. */
     HashToken,
+    /** Only the special JSDoc comment text scanner produces JSDocCommentTextTokes. One of these tokens spans all text after a tag comment's start and before the next @ */
+    JSDocCommentTextToken,

```

Comment:
Should this be internal?

Reply:
Probably. I didn't think you could mark elements of an enum internal.